app/uk/gov/hmrc/helplinefrontend/config/AppConfig.scala

```diff
@@ -192,6 +192,8 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
 
   lazy val platformAnalyticsUrl = servicesConfig.baseUrl("platform-analytics")
 
+  lazy val findYourNationalInsuranceNumberFrontendUrl = servicesConfig.baseUrl("find-your-national-insurance-number-frontend")
```

Comment:
you might find that this needs to be  similar to the syntax below
```suggestion
  lazy val findYourNationalInsuranceNumberFrontendUrl = servicesConfig.getConfString("find-your-national-insurance-number-host")
  // value in applicaion.conf 
 find-your-national-insurance-number-host = "http://localhost:9000"
  // an app config base pr needs to be raised to set this to:
   find-your-national-insurance-number-host = ""
```
Its a common trap that even I fall into

- using base url sets the syntax protocol:host:port
- this means that when you get into QA/Staging/Production you would need to set the host to
--- host = "qa.tax.service.gov.uk"
--- host = "staging.tax.service.gov.uk"
--- host = "tax.service.gov.uk"
I think the port is forgiving but I think I might even be wrong about that and the url doesnt construct correctly due to the requirement of it being port 443
- base url is normally used for backend services due to always needing to provide the host port and protocol for protected calls

Reply:
it also means that colons would remain i believe if you app-config-base override all 3 values you've added in application.conf